library/blivet.py

```diff
@@ -145,15 +148,72 @@ class BlivetAnsibleError(Exception):
     pass
 
 
-class BlivetVolume(object):
+class BlivetBase(object):
```

Comment:
do you need unit tests for these changes?  if so, they can be added later after the 15th